test/test_sparse_csr.py

```diff
@@ -303,9 +303,7 @@ def test_empty_errors(self, layout, device, dtype):
     @dtypes(*all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16))
     def test_clone(self, layout, device, dtype):
         for sparse in self.generate_simple_inputs(
-                layout, device=device, dtype=dtype, index_dtype=torch.int32,
-                # Temporarily disable testing batched block tensors:
-                enable_batched=layout in {torch.sparse_csr, torch.sparse_csc}):
+                layout, device=device, dtype=dtype, index_dtype=torch.int32):
             cloned_sparse = sparse.clone()
```

Comment:
Let's probably test inputs with rectangular block sizes?

Reply:
Rectangular block sizes and many other special cases are actually tested thoroughly via
https://github.com/pytorch/pytorch/blob/1abc4b5e2e6659e22cfb1ab0ae9be589ea06a79d/test/test_sparse.py#L4054
that uses `assertEqual` which triggers `select_copy` calls in
https://github.com/pytorch/pytorch/blob/1abc4b5e2e6659e22cfb1ab0ae9be589ea06a79d/torch/testing/_internal/common_utils.py#L1863

Reply:
However, it is a good idea to have explicit tests for select.

Reply:
I agree, I still do not get why index is divided by blocksize[0] even though it might point to a dimensions that is tiled with blocksize[1]...

Reply:
Tests are now added for `torch.select_copy`.